src/FMBot.Bot/Builders/FriendBuilders.cs

```diff
@@ -388,4 +389,56 @@ public async Task<ResponseModel> RemoveFriendsAsync(ContextModel context, string
 
         return response;
     }
+
+    public async Task<ResponseModel> FriendedAsync(ContextModel context)
+    {
+        var response = new ResponseModel
+        {
+            ResponseType = ResponseType.Embed,
+        };
+
+        if (context.DiscordGuild != null)
+        {
+            response.Embed.WithDescription("This command is only supported in DM");
+            response.CommandResponse = CommandResponse.OnlySupportedInDm;
+            return response;
+        }
+
+        var friended = await this._friendsService.GetFriendedAsync(context.ContextUser.UserId);
+
+        if (friended?.Any() != true)
+        {
+            response.Embed.WithDescription("It doesn't seem like anyone's added you as a friend yet.");
+            response.CommandResponse = CommandResponse.NotFound;
+            return response;
+        }
+
+        response.EmbedAuthor.WithName("People who have added you as a friend in .fmbot");
+
+        var pages = new List<PageBuilder>();
+
+        var friendedPages = friended.ChunkBy(10);
+        var counter = 1;
+        var pageCounter = 1;
+        foreach (var friendedPage in friendedPages)
+        {
+            var friendedPageString = new StringBuilder();
+
+            foreach (var friend in friendedPage)
+            {
+                friendedPageString.AppendLine($"{counter}. *[{friend.User.UserNameLastFM}]({LastfmUrlExtensions.GetUserUrl(friend.User.UserNameLastFM)})*");
```

Comment:
You're using the `User` object here, but that is the user themselves. You can use `FriendUser` instead

Reply:
It's difficult to reason about this. I'll call the user who requested this command the requester.

 `GetFriendedAsync` should return a list of `Friend` who has the requester as the `FriendUser`, because someone at some point has run `.addfriends` on the requester. Thus, the user who is set in `User` should be the person who at some point ran `.addfriends` and not the requester themselves. It's a bit backwards (because this is essentially a reverse lookup, haha), but I hope I've got things the right way here.

Reply:
Oh my bad. Yes, you're completely right.